monkey/infection_monkey/monkey.py

```diff
@@ -117,6 +122,20 @@ def _get_arguments(args):
 
         return opts
 
+    def _get_server(self):
+        servers_iterator = (s for s in self._opts.servers)
+        server = find_server(servers_iterator)
+        if server:
+            logger.debug(f"Default server set to: {server}")
+        else:
+            # TODO: Exit here
```

Comment:
You don't need to exit. `main.py` will catch and log this exception, call `monkey.cleanup()`, and exit.